packages/eas-cli/src/commandUtils/EasCommand.ts

```diff
@@ -66,4 +73,50 @@ export default abstract class EasCommand extends Command {
       setVcsClient(new GitClient());
     }
   }
+
+  private async ensureEasCliIsNotInDependenciesAsync(projectDir: string): Promise<void> {
+    let printCliVersionWarning = false;
+
+    const consoleWarn = (msg?: string): void => {
+      if (msg) {
+        // eslint-disable-next-line no-console
+        console.warn(chalk.yellow(msg));
+      } else {
+        // eslint-disable-next-line no-console
+        console.warn();
+      }
+    };
+
+    if (await this.isEasCliInDependenciesAsync(projectDir)) {
+      printCliVersionWarning = true;
+      consoleWarn(`${chalk.bold('eas-cli')} is added to the project dependencies.`);
+    }
+
+    const maybeRepoRoot = PackageManagerUtils.findWorkspaceRoot(projectDir) ?? projectDir;
+    if (maybeRepoRoot !== projectDir && (await this.isEasCliInDependenciesAsync(maybeRepoRoot))) {
+      printCliVersionWarning = true;
+      consoleWarn(`${chalk.bold('eas-cli')} is added to the monorepo dependencies.`);
+    }
+
+    if (printCliVersionWarning) {
+      consoleWarn(
+        `It's recommended to use the ${chalk.bold(
+          '"cli.version"'
+        )} field in eas.json instead. Use it to enforce the ${chalk.bold(
+          'eas-cli'
+        )} version for your project.`
```

Comment:
```suggestion
        )} version for your project.`
```

let's add a "Learn more" link here too and link out to the eas-cli readme - https://github.com/expo/eas-cli#installation -- and under this installation header let's add a note about using the `cli` key to enforce the version and explain that the package should be installed globally or used through `npx`